README.md

```diff
@@ -52,7 +52,6 @@ have enabled by default.
 To disable a rule, set its value to `null`.
```

Comment:
FYI, You can use `false` now as well since people were assuming that would work too